### PR TITLE
Don't include PR's assigned to owner

### PR DESCRIPTION
**Pull request by PragmaticCypher**
PR's which are assigned to their owner signal they are a Work In
  Progress.

### CODE REVIEW

**Comment by optiopay-backend-helper** [comment]
Pull request seem to be stale, assigning @taheris as the responsible developer.

File: main.go [hunk 1]
```diff
@@ -134,6 +134,17 @@ func stalePullRequests(staleTime time.Duration) (stale []Issue, err error) {
 		}
 		stale = append(stale, issue)
 	}
+
+	// Dev's assign PR's to themselves to signal it is not ready for
+	// being merged.
+	var assignedToOthers []Issue
+	for _, issue := range stale {
+		if issue.Assignee.Login != issue.User.Login {
```
Comment (taheris) [inline main.go]: why not put this check in `range issues` above?

Comment (PragmaticCypher) [inline main.go]: clarity, but theres already a mix of concepts in that loop so I moved it there.

File: main.go [hunk 2]
```diff
@@ -129,9 +129,15 @@ func stalePullRequests(staleTime time.Duration) (stale []Issue, err error) {
 		if !issue.isPullRequest() {
 			continue
 		}
+		if issue.Assignee.Login != issue.User.Login {
```
Comment (taheris) [inline main.go]: I was thinking:

```
if issue.Assignee.Login == issue.user.Login {
  continue
}
```

Comment (PragmaticCypher) [inline main.go]: phrack... fixed

**Pull request merged**